corgi/tasks/monitoring.py

```diff
@@ -64,7 +63,8 @@ def upsert_interval_task(module, task, hours=None, minutes=None):
     upsert_cron_task("brew", "fetch_unprocessed_brew_tag_relations", hour=6, minute=0)
     upsert_cron_task("yum", "load_yum_repositories", hour=7, minute=0)
     upsert_cron_task("yum", "fetch_unprocessed_yum_relations", hour=8, minute=0)
-    upsert_cron_task("monitoring", "email_failed_tasks", hour=10, minute=45)
+    upsert_cron_task("pulp", "fetch_unprocessed_cdn_relations", hour=9, minute=0)
+    upsert_cron_task("monitoring", "email_failed_tasks", hour=11, minute=45)
```

Comment:
What would happen if we made only this change to process CDN relations daily, without any of the other logic, and switched created_since to 1? AKA process only relations created in the last day, every 1 day. Would that reduce the number of tasks being processed enough to avoid these errors?

Reply:
Discussed in a meeting: No, this won't help. We create a bunch of CDN relations all at once, so whether we look at all relations created in the past day or past week, we still have way too many to process.